packages/create-app/templates/default-app/package.json.hbs

```diff
@@ -20,7 +20,6 @@
     "lint": "backstage-cli repo lint --since origin/{{defaultBranch}}",
     "lint:all": "backstage-cli repo lint",
     "prettier:check": "prettier --check .",
-    "create-plugin": "backstage-cli create-plugin --scope internal",
     "new": "backstage-cli new --scope internal"
```

Comment:
The `new` command is the replacement of `create-` style commands. I have not seen any documentation as such except the cli docs.

Reply:
Yep, `new` is the new `create-plugin`. It's not documented anywhere yet because we wanted to add it to apps first and then start pointing towards it later. Updating the documentation is what we want to do in this PR.